%2345678901234567890123456789012345678901234567890123456789012345678901234567890
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% MAURICIO ESGUERRA NEIRA
% Ph. D. Thesis
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\chapter{RNA Base Steps}
\label{basesteps} 
\bibliographystyle{nar}
The problem of classification of the space of conformations
%configurations?
of RNA  is not  new, see for  example, Olson  1972 \cite{olson1_1972},
Saenger     1984     \cite{saenger1984},     and    Gautheret     1993
\cite{gautheret1993}.  This  problem had only been addressed  by a few
researchers  before the  turn of  the twenty  first century,  now this
situation is changing rapidly. The reason for this fast change came in
the year 2000, when a vast amount of RNA structural information became
available  due the  elucidation  of  the structure  of  the 30S  small
ribosomal  subunit  of   \textit{Thermus  thermophilus},  a  bacterial
ribosome  \cite{wimberly2000,  schluenzen2000},   and  the  50S  large
ribosomal subunit of \textit{Haloarcula marismortui}, an archaeal
%\footnote{I emphasize the  phylogeny of rRNA's here since  there is an
%  ongoing discussion among biologists on whether archaea are closer to
%  prokaryotes, or to eukaryotes.}
ribosome \cite{ban2000}.
\begin{figure}[H]
\centering
\includegraphics[scale=0.38]{Chapter2/rna2000_2009copy.png}
\caption{\textbf{Left:}  Total number of  RNA bases  added to  the PDB
  database   between  2000   and   2010  (Exponential   fit  line   in
  blue). \textbf{Right:} Total number  of RNA structures solved yearly
  by X-Ray crystallography between 2000 and 2010 (Exponential fit line
  in red).}
\label{fig:rnainpdb}
\end{figure}

\noindent Between  1978 and  2000 a total  of 116 RNA  structures with
resolution greater than 3.5\AA,  and comprising around 5500 nucleotide
bases are found  in the Protein Data Bank (PDB),  and between 2000 and
today  a  total of  931  RNA\index{RNA}  structures comprising  491158
nucleotide bases are found.  That  is, the increase in information due
to  the  solution of  large  RNA structures  is  about  two orders  of
magnitude as pointed out  by Noller \cite{noller2005}.  Looking at the
growth  of RNA  structural information  from 2000  until today,  it is
clear that  both the total number  of RNA structures  deposited to the
PDB, and the total number  of nucleotide bases in these structures, is
growing in an exponential way (as  can be seen by the exponential fits
in  Figure~\ref{fig:rnainpdb}).   It's  important  to note  that  such
growth comes mainly from ribosomal structures which contain 88 percent
of all RNA  bases in the PDB.  So, even  though structural interest in
RNA is  growing since ribosomal  structures became available  in 2000,
and several  Nobel prizes  have been awarded  for work in  this field,
along  with  the  exciting  possibilities  of  deciphering  large  RNA
\cite{weinberg2009}  structures  other than  the  ribosome, still  the
growth of  the RNA structural  field is far  from that of  proteins if
weighed by  the growth in  diversity of RNA structural  information in
the past decade. At the present time if we look at the distribution of
RNA  sizes   counted  by   number  of  bases,   as  can  be   seen  in
Figure~\ref{fig:rnaranges}  it's clear  that there  are  great patches
where there are no RNA  structures whatsoever, roughly between 600 and
1400 bases  and between  1800 and 2700  bases. The area  of non-coding
RNA's holds great promise for  finding structured RNA's in such length
ranges    as    has     recently    been    suggested    by    Breaker
\cite{weinberg2009}.  A representative  example of  the characteristic
ranges of RNA  structures available to date in the PDB  can be seen in
Table~\ref{tab:rnarange}  for structures  larger than  300  bases.  An
interesting comparison between the  total number of structures of RNA,
protein, dna, and nucleic acid plus protein, available at the PDB from
the   seventies    until   today    can   be   seen    in   Supplement
Figure~\ref{fig:allpolypdb}.

\begin{figure}[H]
\centering
\includegraphics[scale=0.60, angle=270]{Chapter2/histogram.png}
\caption{Frequency of  nucleotide bases in RNA molecules  found in the
  PDB classified by  the size of RNA molecules. We  define the size as
  the total number of nucleotide bases present per molecule.}
  \label{fig:rnaranges}
\end{figure}
%The table and Figure 1.1 come from downloading all structures from
%2000, until today that have RNA in the pdb and that have a resolution
%better that 3.0A, they are 460 structures.

\noindent The analysis of  RNA conformational information contained in
RNA structural  data can be  divided into three main  perspectives: an
atom based perspective; a bond  based perspective; and a third, as yet
unexplored  to our  knowledge, rigid-body  based perspective.   In the
atom  based perspective,  either  direct comparison  of backbone  atom
positions is  made \cite{reijmers2001},  or a comparison  of distances
between a  reduced set  of atoms taken  from the  nucleotide backbone,
sugar,  and  base \cite{sykes2005}.   The  bond  based perspective  is
divided  into   three  main   categories;  the  first   considers  the
consecutive covalent bonds in the RNA backbone and the glycosidic bond
between the sugar  and base, that is, six  backbone torsion angles and
one   glycosidic   torsion   angle   \cite{reijmers2001,   murray2003,
  hershkovitz2003,  schneider2004, hershkovitz2006};  or alternatively
the  pseudo-bonds between consecutive  P and  C4$^{\rm{\prime}}$ atoms
and   the  resulting   pseudo-torsion  angles   $\eta$   and  $\theta$
\cite{olson1_1972,               duarte1998,               duarte2003,
  wadley2007} \footnote{Previously the pseudotorsion angles $\eta$ and
  $\theta$    were    given     the    names    $\omega_{\nu}$,    and
  $\omega_{\nu\rm{\prime}}$.\cite{malathi1985}}.   The  third category
considers the networks of  horizontal hydrogen bonding patterns coming
from  a definition of  interacting edge  boundaries in  the nucleotide
bases \cite{westhof2000, leontis2002, leontis2006}. In this chapter we
study the rigid body based perspective using clustering analysis.
\begin{table}[htbp]
\begin{center}
{\small
\begin{tabular}{c|p{5cm}|c|c|c}
\hline
\bf{PDBID} & \bf{Structure Name} & \bf{Phylogenetic Group} & \bf{Number of bases} & \bf{Year} \\ \hline
1l8v & Mutant of P4-P6 Domain of Group I Intron & Eukaryote & 314 & 2002 \\ \hline
3igi & Group II Intron & Bacteria & 395 & 2009 \\ \hline
1fg0 & Central Loop in Domain V of 23S rRNA & Archaea & 499 & 2000 \\ \hline
2nz4 & GlmS Ribozyme & Eukaryote & 604 & 2006 \\ \hline
1xmq & 30S rRNA & Bacteria & 1522 & 2004 \\ \hline
1ffk & 50S rRNA Subunit & Archaea & 2828 & 2000 \\ \hline
\end{tabular}
}
\caption{Some large  RNA structures  ($>$300 bases) elucidated  in the
  last decade.}
\end{center}
\label{tab:rnarange}
\end{table}

\section{Consensus Clustering of Single Stranded Base Step Parameters}
To  our  knowledge there  has  been  no  classification of  rigid-body
base-step parameters  for RNA structures available from the PDB.  It is
important  to note  here that  in  crystal structures,  RNA bases  are
determined more  accurately than backbone torsion angles,  as has been
shown by Richardson  and collaborators from analysis of  van der Waals
steric    clashes.    This    can    be   seen    more   clearly    in
Figure~\ref{fig:murray},    reproduced    from    Richardson's    work
\cite{murray2003}, where the red and orange dots in the backbone atoms
region denote steric clashes and the green and yellow dots in the base
atoms region  denote very good  agreement with expected van  der Waals
distances.
\begin{figure}[htbp]
 \centering
 \includegraphics[scale=0.5]{Chapter2/murray2003.png}
 \caption{Figure taken from  Richardson et al. \cite{murray2003} where
 the  blue and  green dots  in  a) mean  very accurate  van der  Waals
 distances, and  in b)  the red and  orange dots mean  steric clashes,
 that is, distances outside the acceptable van der Waals range.}
 \label{fig:murray}
\end{figure}

\subsection{Combining Fourier Averaging Results and Clustering Analysis}
Using  the  coordinate  files  of  20  rRNA  structures  provided  by
Schneider  at  al.\cite{schneider2004}  we  used  standard  clustering
analysis  (CA)  techniques (see  Appendix  A)  to  classify a  set  of
non-ARNA base-steps using, rather  than the more common torsion angles
space, the  base-step parameters  space, that is,  three translational
parameters  (Shift   $D_x$,  Slide  $D_y$,  Rise   $D_z$),  and  three
rotational  parameters  (Tilt $\tau$,  Roll  $\rho$, Twist  $\omega$),
which we describe with the hexaparametric vector $\nu$:
\begin{gather}
 \nu = (D_x, D_y, D_z, \tau, \rho, \omega)
\end{gather}
The    results    illustrated    in    the   dendrogram    shown    in
Figure~\ref{fig:eucl_cons}  and  whose  corresponding  structures  are
shown in  Figures~\ref{fig:nonAclus}~\ref{fig:steps2} were obtained by
performing  clustering   analysis  and  consensus   clustering  on  20
structures provided by  Schneider et al.  \cite{schneider2004}.  These
twenty  structures  were  obtained  by Schneider  applying  a  Fourier
averaging technique, and lexicographical clustering, to torsion angles
of 23S rRNA.   The methodology we used follows that  used by others to
recover  the  periodic   table  classification  from  multidimensional
property vectors for elements \cite{restrepo2004, restrepo2006}.
\begin{figure}[htbp]
 \centering
\includegraphics[angle=90, scale=0.6]{Chapter2/eucli_cons_nonA-RNA.png}
\caption{Dendrogram showing the results  of consensus clustering of 20
non-Atype  rRNA  dinucleotides   according  to  their  hexadimensional
base-step parameter vectors.}
 \label{fig:eucl_cons}
\end{figure}

\begin{figure}[htbp]
 \centering
\includegraphics[angle=90, scale=0.5]{Chapter2/collageb.png}
 \caption{RNA dinucleotide structures  organized by clusters obtained
   from  consensus  clustering   of  their  hexadimensional  base-step
   parameter  vectors.  The  structures  have  been  centered  on  the
   reference frame of  the first step, that is,  the adenine base, and
   the minor  groove face of  the rigid block parameter  associated to
   adenine is facing the viewer.}
 \label{fig:nonAclus}
\end{figure}

In Figures~\ref{fig:nonAclus}, and~\ref{fig:steps2}  we see that Group
I contains structure {1}  with base-plane normals pointing in opposite
directions, Group II  includes extended conformations with neighboring
bases roughly  parallel but  not stacked and  is formed  by structures
{15, 16, 10, 14}, Group  III also contains extended conformations with
bases perpendicular to one another  and is formed by structures {8, 9,
  17}, Group IV  {18, 19, 20, 13, 11,  12, 5, 3, 6, 7,  2, 4} contains
four major subgroups:  (a) structures {2, 4} which  are unstacked with
bases neither parallel nor  perpendicular; (b) structures {18, 19, 20}
which closely relate  to A-RNA; (c) structures {11,  12, 13} which are
unstacked and  have parallel  bases; and (d)  structures {3, 5,  6, 7}
which are also unstacked and have parallel bases. We also see in Group
IV that  the conformers  in subgroups  IV (c) and  IV (d)  are closely
related, and that  the dimers in these two  subgroups are more closely
related to those in subgroup IV (b) than to those in subgroup IV (a).

To account for the representation of the groups obtained by clustering
in   the  23S   subunit  of   the  ribosome   we  have   computed  the
root-mean-square deviation (RMSD)  between the average step parameters
of the structures composing each group, and the step-parameters of the
2753 steps present in 23S.  That is, for each group we have obtained a
set of RMSD  values which have been plotted as  histograms as shown in
Figures~\ref{fig:histo1}, and~\ref{fig:histo2}.   The results are also
summarized in  Table~\ref{tab:nonA}, where we  can see that  they only
constitute 31\% of the total amount of steps in the 23S subunit of the
ribosome.   We  used  a  cutoff  of 10  \AA  \footnote{We  retain  the
  traditional unit  of Angstroms  to refers to  our RMSD's, but  it is
  important  to note that  since we  are not  refering to  an all-atom
  model such unit does not  have a direct physical meaning.} to select
the structures which belong to each group, based on visual analysis of
superimposed reconstructed structures. For example, for Group I; if we
reconstruct the ribosomal steps with an RMSD of 10 \AA or less, we get
the      figure      shown     in      the      left     panel      of
Figure~\ref{fig:superimpose}. But  if we  reconstruct with the  set of
structures with an RMSD of 15 \AA or less we start getting structures,
which after  being superimposed based  on the reference frames  of the
first base  are clearly not related to  that group, as can  be seen in
the right panel of Figure~\ref{fig:superimpose}.

We have also  noticed that the starting structures  kindly provided to
us by Dr. Berman,  have a large rise in the case  of A-RNA, that is, a
value of  4.39 \AA, which is  larger than the 3.30  \AA value obtained
for  the "classical"  A-RNA  structure from  Arnott and  collaborators
\cite{arnott1973}. This might have  a significant effect on the amount
of structures which can be grouped under the A-RNA like group.

Because of not getting a good representation of the total diversity of
base-steps  in the  23S  subunit of  the  ribosome, we  have opted  to
perform an  analysis based fully  on base-step parameters.  We believe
that the reason  for such poor representation is due  to the mixing of
Fourier averaging for backbones, and  the base-step perspective.
%% or it could also be  related to the fact that we  are ignoring the
%% remaining A-RNA related set of structures in Berman and
%% collaborators analysis.

\begin{table}[htbp]
\begin{center}
{\footnotesize
\begin{tabular}{c|r|c}
\hline
\bf{Group} & \bf{Percentage} & \bf{Number of Base-Steps}\\ \hline
I   & 0.11  & 3   \\ \hline
II  & 0.18  & 5  \\ \hline
III & 0.04  & 1  \\ \hline
IVa & 0.36  & 1  \\ \hline
IVb & 29.31 & 807 \\ \hline
IVc & 0.33  & 9  \\ \hline
IVd & 1.27  & 35 \\ \hline \hline
Total & 31.28 & 861   \\ \hline
\end{tabular}
}
\caption{Number of base-steps  with RMSD values less than  or equal to
  10 \AA ~between the reference  base-step vectors from the four groups
  of  non-A-type  RNA  dinucleotide  conformations and  all  base-step
  vectors found in the  23S strand of \textit{Haloarcula marismortui}.
  The  percentage  is calculated  with  respect  to  a total  of  2753
  base-steps  present in  the  23S chain  of  the 50S  subunit of  the
  ribosome.}
\label{tab:nonA}
\end{center}
\end{table}

\begin{figure}[htbp]
 \centering
\includegraphics[angle=90, scale=0.6]{Chapter2/RMSDschneider1.png}
\caption{Root mean square deviation of the main four groups show in
  Figure~\ref{fig:nonAclus}. The color of the histograms is the same
  as that of the boxes surrounding the structures of
  Figure~\ref{fig:nonAclus}}
 \label{fig:histo1}
\end{figure}

\begin{figure}[htbp]
 \centering
\includegraphics[angle=90, scale=0.6]{Chapter2/RMSDschneider2.png}
\caption{Root  mean  square  deviation  histograms for  the  subgroups
  present in group IV.  Since subgroup IVb is composed of A-RNA like
  conformations we see in the upper left histogram that the highest
  proportion of small RMSD values belongs to this group.}
 \label{fig:histo2}
\end{figure}

\begin{figure}[htp]
 \centering
\includegraphics[angle=0, scale=0.3]{Chapter2/G1at10_15.png}
\caption{\textbf{Left}: With an RMSD cutoff of less than or equal to
  10 \AA, we identify three steps from the 23S subunit. \textbf{Right}:
  With an RMSD less than or equal to 15 \AA ~we get a total of seven
  structures, where we clearly see that three of them are farther from
the original Group I main structure of Figure~\ref{fig:nonAclus}}
 \label{fig:superimpose}
\end{figure}

\subsection{Selection of a Clustering Methodology}
In  order to  analyze  our  dataset of  base-step  parameters we  have
decided  to  use  clustering  analysis methods.   Clustering  analysis
methods can  be broadly classiffied  in two main categories,  that is,
they  can be  partitional or  hierarchical.  In  either case  the main
problem  one faces  for classification  purposes is  that  of deciding
which  is the  optimal number  of hierarchies  or partitions  that the
analyzed  data is split  into.  To  obtain a  criteria for  an optimal
number of  clusters, and also to  decide which method  might be better
for  our  dataset,  we  have  used two  types  of  cluster  validation
techniques.   They  are  known  as  internal  measures  and  stability
measures. Full detail on the  definition of such measures are provided
in  \cite{handl2005, brock2008}.  To  perform the  validation analysis
mentioned above  we used a  cluster validation package  implemented in
the  R  \cite{rcite}   statistical  analysis  program  called  clValid
\cite{brock2008}.

In  Figures~\ref{fig:internal} and~\ref{fig:stability} we  present the
results for  internal and  stability validation results  exploring the
same  dataset of  base-step  paramaters  for the  23S  subunit of  the
ribosome that we've used  before. In the clustering anlysis literature
it's  customary  to use  the  variable $k$  to  define  the number  of
clusters  and  we  will  use  variable  $k$  in  that  sense  in  what
follows.

Our analysis computed  the validation scores for a  number of clusters
ranging from $k=2$, up  to $k=80$ clusters, and evaluated hierarchical
methods (hierarchical,  diana), and partitional  methods (kmeans, pam,
som,  sota).  The  connectivity  measure must  be  minimized, and  the
average silhouette width (silhoutte) and dunn index must be maximized.
With   this   in  mind,   we   see   that   the  method   labeled   as
hierarchical\footnote{The hierarchical  label refers precisely  to the
  agglomerative (bottom-up)  technique, the euclidean  metric, and the
  average method.}  performs better in connectivity and dunn index for
the whole range,  and it is also the best  performer in silhoutte from
$k=12$ onwards.

In the  case of the  stability measures it  is important to  note here
that mainly these measures are  well suited for highly correlated data
sets, therefore they  are not very indicative for  our data set, which
is correlated  in shift and twist, as  can be seen from  the values on
the   upper  right   corner   of  the   pairs   scatteplot  shown   in
Figure~\ref{fig:pairsnoarna}.    We  include  these cluster  stability
measures for completeness.

The stability measurements  we have computed are read  as being better
the smaller their values, of  these we have quantified three measures,
that  is, the  average proportion  of non-overlap  (APN),  the average
distance (AD), and the average distance between means (ADM). The
details of  such measures are given  in Brock et al. \cite{brock2008}.
As  seen  in  Figure~\ref{fig:stability}  the  method  with  the  best
stability measures is  sota for APN, and ADM, almost  in all the range
until it reaches a number of clusters of around 70. For the AD measure
the best performers are pam and  sota in the whole range.  Notice that
the hierarchical method  follows the same trend as  the other methods,
and that in  general, apart from the APN measure  and the sota method,
all methods have a similar behaviour due to the fact that our data set
is not highly  correlated, that is, it cannot be  split into say, two,
three, or four, principal components.

In all cases  we also see that the best overall  number of clusters is
two, which is not surprising since we haven't filtered out A-RNA
structures from our  data set, leaving two main  groups; that of A-RNA
type base-steps, and those which are not A-RNA like.

\begin{figure}
 \centering
\includegraphics[angle=0, scale=0.38]{Chapter2/STval_int.png}
\caption{Cluster validity scores for internal measures. Notice how the
  hierarchical method, labeled as 1 in black color,
  behaves better for the whole range of Connectivity (smaller values)
  and Dunn (higher values),
  and it also outperforms all others after $k=12$ for Silhoutte
  (higher values) scores.}
 \label{fig:internal}
\end{figure}

\begin{figure}
 \centering
\includegraphics[angle=0, scale=0.38]{Chapter2/STval_sta.png}
\caption{Cluster validity scores for stability measures.}
 \label{fig:stability}
\end{figure}

We focus  our attention in  the group of  structures which are  not so
closely related to A-RNA. Therefore we have filtered them out based on
Figure~\ref{fig:dormsd}, and end out with a data set of 797 (about 29\%
of the  total number of  steps) base-step parameters whose  values are
greater than an RMSD of 18  \AA.  These RMSD values have been computed
between the base-step parameters of 23S RNA and the standard base-step
parameter    values   derived    from    Arnott   and    collaborators
\cite{arnott1973} work. Standard base-step parameter values are provided in 
table~\ref{tab:conformations} for common double-stranded conformations
of RNA, and DNA.

\begin{figure}
 \centering
\includegraphics[angle=0, scale=0.4]{Chapter2/dormsd.png}
\caption{RMSD values between base-step parameters of the 23S subunit of
  ribosomal RNA and the standard base-step parameters derived from
  Arnott and collaborators \cite{arnott1973} work.}
 \label{fig:dormsd}
\end{figure}

\begin{table}[htbp]
\begin{center}
{\small
\begin{tabular}{p{2cm}|c|c|c|c|c|c|c}
\hline
\textbf{Structure Name} & Shift ($D_x$) & Slide ($D_y$) & Rise ($D_z$) & Tilt
($\tau$) & Roll ($\rho$) & Twist ($\Omega$) & \textbf{Reference} \\ \hline
A-DNA & 0.36 & -1.39 & 3.29 & 2.46 & 12.50 & 30.19 & \\ \hline
B-DNA & 0.44 & 0.47 & 3.33 & 4.63 & 1.77 & 35.67 & \\ \hline
A-RNA & -0.08 & -1.48 & 3.30 & -0.43 & 8.64 & 31.57 & Arnott \\ \hline
A'-RNA & 0.05 & -1.88 & 3.39 & -0.12 & 5.43 & 29.52 & Arnott \\ \hline
AII-RNA & 1.01 & -2.52 & 3.33 & 2.94 & 9.75 & 25.12 & Schneider \\ \hline
\end{tabular}
}
\caption{Base step parameters for common DNA and RNA
  conformations. The base-step parameters are computed for
  a single-stranded base-step rather than a double-stranded base-pair step.}
\end{center}
\label{tab:conformations}
\end{table}

With  the filtered dataset,  which we  will refer  to as  non-ARNA
dataset, we  have again repeated the cluster validation analysis for
internal measures as can be seen in Figure~\ref{fig:noarna}. From this
analysis we see again that the best method for clustering our dataset
is the hierarchical one. The Dunn index, which
works under the idea of finding the best possible separation and
compactness between clusters, shows us that the optimal number of
clusters is $k=67$. The other two indices show, as for the case where
the whole dataset was analyzed, that the optimal number of clusters is
two.
We selected the 67 clusters given by the hierarchical method, and took
their corresponding step-parameter values to reconstruct the
dinucleotide step structures using 3DNA. In Figure~\ref{fig:noarnak67}
we see the first seventeen groups which contained ten or more
structures on them. We see that they are quite 


Statistics for the Purine and Pyrimidine content
can be seen in Supplementary table ~


reconstruted dinucleotide groups in block
representation. From this collage of structures we see that



\begin{table}[htbp]
\begin{center}
{\small
\begin{tabular}{p{2cm}|c|c|c|c|c|c|c}
\hline


\end{tabular}
}
\caption{Base step parameters for common DNA and RNA
  conformations. The base-step parameters are computed for
  a single-stranded base-step rather than a double-stranded base-pair step.}
\end{center}
\label{tab:conformations}
\end{table}






\begin{figure}
\centering
\includegraphics[angle=0, scale=0.4]{Chapter2/pairsnoarna.png}
\caption{. 
  }
\label{fig:pairnoarna}
\end{figure}



\bibliography{biblio}

