\chapter{RNA Motifs}
\label{motifs} 
\bibliographystyle{nar}
As  mentioned  in  pages 24  and  25  in  Chapter  2 the  most  common
perspectives for RNA motif recognition  and discovery are the atom and
bond  based ones,  the  rigid-body-based perspective  has been  rather
unexplored.  The two  main questions that we would  like to address in
this context are:

\begin{enumerate}
\item{Can  the geometric rigid-block  description of  base-pairing and
  base-stacking solve the problem of defining RNA structural motifs?}
\item{Can other  quantities derived from the 3DNA  software package be
  used to make and automatic search for known motifs, for example, the
  GNRA tetraloop motif, and perhaps find unknown motifs?}
\end{enumerate}

We have started with the  second question and have chosen as workhorse
the  well  known  GNRA  tetraloop  motif.  We  have  also  used  other
quantities  (e.g.  endocyclic  and  exocyclic base-overlaps)  obtained
with  the  3DNA \cite{lu2003,  lu2008b}  package,  and explored  their
relationship to RNA motifs.

\section{The GNRA Tetraloop}
The GNRA motif  was initially found to be  an important constituent of
the small subunit  of the ribosome using the  technique of comparative
sequence analysis \cite{woese1990}, that is, it was seen that the GNRA
sequence was frequently repeated  among various organisms, and so were
the CUUG  and UNCG  tetralops which  amount to more  than 70\%  of all
tetraloops found in the 16S subunit of ribosomal RNA \cite{woese1990,
  depaul2010}.   The  most  abundant  of  all the  tetraloops  in  the
ribosome  is   GNRA  and   its  structural  stability   was  initially
characterized by NMR studies by  Heus and Pardi \cite {heus1991} which
determined that the loops contain a non-canonical sheared GA base-pair
flanking the  sequence, a hydrogen bond  between a guanine  base and a
phosphate, extensive base stacking, and a hydrogen bond between the OH
of a sugar 2$'$ end and a base \cite{heus1991}.

The description of  the GNRA tetraloop motif is a  typical case of the
problem  of RNA  motif  definition,  for example,  in  the context  of
sequence  alone  a  GNRA  motif  would  be one  which  follows,  in  a
consecutive manner, the GNRA pattern, whereas it has been noticed that
there can be GNRA structures which are not consecutive in sequence but
have  the same  geometric  arrangement of  bases in  three-dimensional
space, and also structures have  the same geometric arrangement as the
GNRA  tetraloop but  do  not  follow the  same  sequence pattern,  for
example, UCAA,  UCAC, CAGA  and CAAC \cite{lemieux2006}.   These other
sequences  which are  geometrically equivalent  to the  GNRA tetraloop
motif  make a  non-canonical base-pair  which is  isosteric to  the GA
base-pair,  that is, the  UA, UC,  CA, and  CC base-pairs  closing the
tetraloops are isosteric to the GA pair \cite{lemieux2006}.

Studies  have   also  been   carried  out  using   molecular  dynamics
simulations to explore the  conformational space of the GNRA tetraloop
and the relation  of the conformational states found  in their results
to existing  X-Ray and  NMR structures available  at the  protein data
bank \cite{sorin2002, depaul2010}. Also other MD studies have used the
the  well  known GNRA  motif conformation  as starting  point for
simulation of  other tetraloops which  do not necessarily turn  out to
retain a GNRA-like three-dimensional structure \cite{srinivasan1998}.

\subsection{GNRA Motif Search Program}
We have  devised a simple algorithm to search for GNRA  motifs based on
their base  step parameters.  The  algorithm is illustrated  in Figure
\ref{fig:getGNRA}.

\begin{figure}
\centering
\includegraphics[angle=0, scale=0.4]{Chapter5/getGNRA.png}
\caption{Simple algorithm  for GNRA motif  finding based on  base step
  parameters.}
\label{fig:getGNRA}
\end{figure}
  
The algorithm allows for any seed to  be used, but for now we only use
the step parameters  of the GNRA motif as seed  for our algorithm.  We
are  currently constructing  a  database of  base-step parameters  for
known motifs, so  that our simple program ``getGNRA''  can be expanded
to include any known motif the  user wishes to localize in a given RNA
structure.

The  algorithm has  been programed  as a  simple, yet  very  fast bash
script  which interfaces  with two  other components,  one  written in
python and the  other written in the statistical  analysis software R.
For example, for  the large subunit of the  ribosome, PDB-ID:1jj2, the
program  takes  22.049  seconds  to  download and  analyze  the  whole
structure. 

The  program, which  we have  called ``getGNRA'',  allows the  user to
query any pdb id. That is, the user only needs to input in the command
line of a UNIX/LINUX terminal the command, ``getGNRA'' followed by the
PDB ID of the RNA molecule of interest. As a result the user obtains a
list composed of residue numbers  corresponding to the location of the
start of the motif in the  structure, and a score which stands for how
close or far  a four nucleotide sequential structure  is from the GNRA
motif seed.   The advantage  of our program  over other  windows based
motif  recognition softwares,  besides from  providing a  new analysis
based on rigid-body parameters, is that it allows for easy integration
of  automated  scripts  for   processing  large  lists  of  known  pdb
structures  without  user  intervention   for  the  analysis  of  every
structure. For  example, in the RNA ontology  consortium (ROC) meeting
of  May,  2009   a  reduced  dataset  of  RNA   structures  found  at:
\url{https://docs.google.com/Doc?id=dhrmkfmn_13ftpbjcgq}    was   made
available to participants with the  purpose of allowing them to search
for RNA  motifs which  would later be  compared between  groups. Using
windows  based  softwares   like  FR3D  \cite{sarver2008}  it's  quite
difficult,  if not  impossible, for  the user  to submit  a  large job
composed  of many  pdb structures  to a  queing system,  or  a cluster
computing server, such task is quite simple using getGNRA.

We  have  taken the  mean  single stranded  base-step
parameters in 14 GNRA tetraloops found  in the 1ffk for the GN, NR, RA
steps and  put them into a  3 by 6 matrix.  The values can  be seen in
table:

We  compute the  sum of  the difference  between the  GNRA  matrix and
sequential 3 by six  matrices resulting from the step-parameter output
of 3DNA. With  this score we can find the GNRA  tetraloops given a pdb
file.

\begin{figure}
\centering 
%\includegraphics[angle=0]{Chapter5/gnra24.png}
\includegraphics[angle=0, scale=2]{Chapter5/gnra24.png}
\caption{A GNRA Tetraloop from the \textit{Azoarcus} group I intron,
  PDB-ID:3iin  \cite{antonioli2010}  found  using our  simple  program
  getGNRA.}
\end{figure}



We have started with just a few, five. The results are shown in:

The  algorithm still  falls for  cases  where the  pdb structures  are
non-sequential and  have large jumps  in sequence, or where  there are
more than one  single stranded chain, for example  in 1ffk since there
are two residues which are numbered 90, one in the 23S, and one in the
5S, then it finds a mismatch on the 23S.


%\section{Canonical "Noise"}
%To be able to say anything about motifs it's crucial to get rid of the
%"noise" which is given by the canonical base-pair steps.
%One would think that perhaps  the X3DNA-Parser of Dr. Yurong Xin could
%help  in the  task, but  then, it  can't, because  it's based  on what
%base-pairs  have  been found,  therefore,  it  tells  me about  single
%stranded interactions,  it doesn't tell  me about bases which  are not
%forming interactions and are alone.

%\subsection{3DNA-Parser}
%We started by using Dr. Yurong Xin's 3DNA-Parser hoping that the
%description of the enclosing base pair in the loop, that is, the
%sheared G$\cdot$A, would have a characteristic signature.
%We found that such is not the case. We know from Major et
%al. \cite{lemieux2006} that there should be at least 21 GNRA tetraloops
%in the 23S subunit of rRNA. We used the G2696 N2697 R2698 A2699
%tetraloop as a seed (as can be seen in Figure 1.1) and found out
%that according to Dr. Xin's helical classification the enclosing G is
%classified as $S_{hq}$ and A is classified as $H_{e}$. 

%We then searched all such instances for G$\cdot$A base-pairs and we
%found seven hits, but none were in fact GNRA tetraloops.

\subsection{Overlap Scores} 
We  clustered the  overlap  values  impossing a  cutoff  of values  of
[1-8]. Since a large amount of overlap values are exactly zero (33\%),
so, without the cutoff the zero values "overshadow" the data.
\begin{figure}[htbp]
\centering 
\includegraphics[angle=0, scale=0.8]{Chapter5/histocompare.png}
\caption{Normalized  histograms showing  the  distribution of  overlap
  values  in the  23S subunit  or \textit{Thermus  Thermophilus} rRNA,
  PDB-ID:1jjk.  In  histogram (a)  all  values  are  included, but  in
  histogram (b) only values greater than zero are included. Notice the
  high preponderance  of zero  values, exactly 897  out of a  total of
  2705.}
\end{figure}
For this case we obtained a ``good'' dendrogram as seen in Figure 1.2.
\begin{figure}[htbp]
\centering 
\includegraphics[angle=90, scale=0.6]{Chapter5/eucli_cons.png}
\caption{Dendrogram for consensus clustering  of overlap scores in the
  ribosome.  Zero values filtered out and remaining data normalized.}
\end{figure}

The next  step in this analysis  will be to find  the structures which
correspond  to this  clusters  and superimpose  and  align them  using
Kabsh's algorithm to be able to determine their RMSD's.

Many people  start their  RNA Motif identification  and classification
algorithms by splitting  RNA structures into what is  helical and what
is not,  and then  finding interactions between  these two  groups. We
believe that  we could do  a similar exercise  with 3DNA by  using the
scalar  product   of  helical  axis  vectors  and   once  helical  and
non-helical regions are  found we might be able to  use 3DNA Parser to
look for characteristic interactions.

%\section{Triplets on RNA (comparison to Laing et al.)}

\section{Conclusions}
We  answer  the questions  at  the begining  of  this  chapter in  the
following way:

\begin{enumerate}
\item{\textbf{Q.}  Can   the  geometric  rigid-block   description  of
  base-pairing  and base-stacking  solve the  problem of  defining RNA
  structural motifs?}
\item{\textbf{A.}  The problem  of defining  RNA structural  motifs is
  clearly  more  complicated  than  what  can  be  understood  by  any
  structural  research  methodology  alone.  We have  shown  that  the
  rigid-body parameter view of RNA  can easily automate the process of
  motif  searches  in RNA  atomic  structures,  and  it helps  in  the
  description   of   motifs,    therefore   we   believe   that   this
  characterization should not be ignored by the community and included
  in ontological efforts such as the ROC one.}
\item{\textbf{Q.} Can we use quantities derived from the 3DNA software
  package to make and automatic search for a known motif, for example,
  the GNRA tetraloop motif, and perhaps find unknown motifs?}
\item{\textbf{A.} Yes. by defining seeds from known motifs we can find
new motifs in  the boundaries of known ones  via base-pairs steps. But
we  can also  do  other,  yet simpler,  not  as ``precise''  searches,
e.g. by clustering  the results of base overlaps.  Other searches have
not  been explored  but could  also be  useful, for  example exploring
helical  parameters like x-displacement,  y-displacement, inclination,
tip.} 
\end{enumerate}

\bibliography{biblio}

